ui/src/components/screens/ErrorPage.js

```diff
@@ -2,11 +2,40 @@ import { Center, Icon, Text, VStack, Box, Button } from '@chakra-ui/react';
 import { Link, useHistory } from 'react-router-dom';
 import { MdErrorOutline } from 'react-icons/md';
 import { AiFillCheckCircle } from 'react-icons/ai';
-import { DateTime } from 'luxon';
+import { useEffect, useState } from 'react';
+import { validateForm } from '../helpers/formsHelpers';
 
 const ErrorPage = (props) => {
   // Add new cases corresponding to the errorPages definition
   const history = useHistory();
+  const [isLoading, setIsLoading] = useState(true);
+  const [formOpenTime, setFormOpenTime] = useState(history.location.state.availableAfter)
+
+  useEffect(() => {
+    const validate = async () => {
+      const { location: { state } } = history;
+      const { user } = props;
+      if (state && state.id) {
+        const result = await validateForm(state.id, user);
+        if (result.data) {
+          history.push(`/forms/${state.id}`);
+          return;
+        } else if (result.pathname && result.pathname !== history.location.pathname) {
+          history.push({
+            pathname: result.pathname,
+            state: result.state
+          });
+          return;
```

Comment:
it will occur when let's say:
- person loads form A (which is unavailable yet)
- error page shows form is unavailable
- admin updates the form details by adding a form open time
- person reloads the page
- error page now shows form open time